app/pages/project/project-navbar.jsx

```diff
@@ -0,0 +1,416 @@
+import { IndexLink, Link } from 'react-router';
+import classnames from 'classnames';
+import SOCIAL_ICONS from '../../lib/social-icons';
+import Thumbnail from '../../components/thumbnail';
+import isAdmin from '../../lib/is-admin';
+
+// constants
+const AVATAR_SIZE = 100;
+
+function Avatar({ projectAvatar }) {
+  return (
+    projectAvatar ?
+      <Thumbnail
+        src={projectAvatar.src}
+        className="avatar"
+        width={AVATAR_SIZE}
+        height={AVATAR_SIZE}
+      />
+      : null
+  );
+}
+
+function ProjectName({ loading, project, translation }) {
+  if (loading) {
+    return <span>Loading…</span>;
+  }
+  if (project.launch_approved) {
+    return (
+      <span>
+        {translation.display_name}
+        <span className="checkmark-stack fa-stack" aria-label="Zooniverse Approved" title="Zooniverse Approved" role="img">
+          <i className="fa fa-circle fa-stack-2x" />
+          <i className="fa fa-check fa-stack-1x checkmark-stack__checkmark" />
+        </span>
+      </span>
+    );
+  }
+  if (project.beta_approved) {
+    return (
+      <div className="project-name">
+        {translation.display_name}
+        <small className="project-name__under-review">Under Review</small>
+      </div>
+    );
+  }
+  return <span>{translation.display_name}</span>;
+}
+
+function HomeTab({ project, projectPath, translation, onClick, children }) {
+  const avatarClasses = classnames('tabbed-content-tab', {
+    '`beta`-approved': project
+  });
+  return (
+    project.redirect ?
+      <a
+        href={project.redirect}
+        className="tabbed-content-tab"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        Visit {translation.display_name}
+      </a> :
+      <IndexLink
+        to={projectPath}
+        activeClassName="active"
+        className={avatarClasses}
+        onClick={onClick}
+      >
+        {children}
+      </IndexLink>
+  );
+}
+
+function AboutTab({ project, projectPath, onClick}) {
+  return (
+    project.redirect ?
+    null :
+    <Link
+      to={`${projectPath}/about`}
+      activeClassName="active"
+      className="tabbed-content-tab"
+      onClick={onClick}
+    >
+      <Translate content="project.nav.about" />
+    </Link>
+  );
+}
+
+function ClassifyLink({ projectPath, workflow, onClick }) {
+  const opacity = {
+    opacity: 0.5
+  };
+  return (
+    workflow ?
+      <Link
+        to={`${projectPath}/classify`}
+        activeClassName="active"
+        className="classify tabbed-content-tab"
+        onClick={onClick}
+      >
+        <Translate content="project.nav.classify" />
+      </Link> :
+      <span
+        className="classify tabbed-content-tab"
+        title="Loading..."
+        style={opacity}
+      >
+        <Translate content="project.nav.classify" />
+      </span>
+  );
+}
+
+function ClassifyTab({ project, projectPath, workflow, onClick }) {
+  function redirectClassifyLink(redirect) {
+    return `${redirect}classify`;
```

Comment:
Does the `redirect` variable here assume that the URL ends with a trailing slash? If yes, we'll want to reimplenent the solution from `getProjectLinks`

This `ClassifyTab()` function is running a bit too deep for me to figure out at a glance what sorta data's being passed, so I'll need to check this on a local test.

Reply:
that's the old navbar file, it shouldn't have been there in the first place, sorry.